src/zone.ts

```diff
@@ -166,6 +172,17 @@ export interface ZoneExportCallback {
   (err: Error | null): void;
 }
 
+type Without<T, K> = {
```

Comment:
This made my brain melt.  Can you add a quick comment letting the unsuspecting javascript developer know what's happening here?

Reply:
Caaan do

Reply:
Yeah, a comment would be great here :+1:

Reply:
Done, PTAL